### ISSUE TITLE
BERT training graph

### ISSUE THREAD

**Issue by jboelaert** [emilienschultz/activetigger#371]
For an improved BERT graph:
- let's make it bigger
- xlim = [0, nepochs]
- start plotting after 1st eval (instead of 2d)

**Comment by jboelaert** [comment]
En voilà une belle courbe !
Attention cependant :
- 8 epochs c'est vraiment beaucoup, normalement on doit pouvoir s'en sortir
avec 3-4, avec donc un learning rate plus haut (ou des batchs plus petits)
- des minibatch totaux de 5x16 = 80 c'est extrême ! sauf à avoir des
centaines de milliers de cas d'entraînement, on ne dépasse généralement pas
un total de 32

Je garde ça en tête pour faire le guide de fine-tuning

**Comment by julesbrion** [comment]
Merci pour les conseils !

**Comment by emilienschultz** [comment]
I improved the visualisation. 

Nevertheless, there is still something odd (related to Jules comment). 

The number of epochs doesn't match the initial parameters. 

I guess it is linked to the way steps are managed here https://github.com/emilienschultz/activetigger/blob/ef86153f4485c88c62fa97ebac0c967c404c6d71/api/activetigger/tasks/train_bert.py#L217 

If @jboelaert you have any idea, I take it.
